v2/devices.go

```diff
@@ -8,6 +8,14 @@ import (
 	"time"
 )
 
+type Devices struct {
```

Comment:
If we're keeping things in the same package, do we really want to use these simpler names for these services?  I suspect we're more likely to run into conflicts with types we want to use for data types within the API.  `DevicesService` or similar?

Reply:
I'm open to qualifying the names somehow, though I don't like `Service` since our API explicitly not service oriented by entity-oriented/RESTful.

Reply:
How about `DevicesResource`, since we're modeling REST resources here?